src/pages/installing-spark.mdx

```diff
@@ -28,4 +28,4 @@ Spark in various states of implementation.
 |---|---|
 | `master` | The Spark Design System and all of its prerequisites are installed. Start building with Spark components right away. |
 | `kitchen-sink` | The Spark Design System and all of its prerequisites are installed. In addition, a test page has been added that includes all the Spark components. This is useful for testing the latest version of Spark. |
-| `without-spark` | This is a boilerplate app without Spark installed. This is useful for following along in our [Getting Started Guides](). |
+| `without-spark` | This is a boilerplate app without Spark installed. This is useful for following along in our [Getting Started Guides](./html-installation). |
```

Comment:
i think you need /installing-spark/html-installation here instead of ./html-installation